README.md

```diff
@@ -5,3 +5,7 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
 ---
+
+## Project Status
+
+[![Reviewed by Hound](http://img.shields.io/badge/Reviewed%20By-Hound-%23a874d1)](https://houndci.com)
```

Comment:
Kindly move the badge to the top of the readme

Reply:
> Look into the comment I dropped and squash your commit I can see four commit instead of one and also follow the commit pattern defined

I have made the requested changes.

Reply:
Nice work done